hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/CryptoQueryHandler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.token;
+
+import com.hedera.node.app.spi.QueryHandler;
+import com.hedera.services.store.models.Account;
+import com.hederahashgraph.api.proto.java.*;
+import java.util.Optional;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * Defines APIs for responding to queries on state. Most of these APIs are defined in
+ * "CryptoService" in the protobuf. Some APIs are defined for the use of other modules, or the
+ * Hedera application. Some queries are paid, some, some are free.
+ */
+public interface CryptoQueryHandler extends QueryHandler {
```

Comment:
Should this be part of an token-service-api module?

Reply:
I think so, once we have such a module. But we don't have one yet...